CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/compatibility/AnalyticSchemaV4UpdateProvider.java

```diff
@@ -76,7 +76,7 @@ protected void schemaUpdate(final StoreGraph graph) {
             final int vertexId = graph.getVertex(vertex);
             final SchemaVertexType oldType = graph.getObjectValue(typeAttribute, vertexId);
 
-            if (typesToUpgrade.containsKey(oldType.getName())) {
+            if (oldType!=null && typesToUpgrade.containsKey(oldType.getName())) {
```

Comment:
I'd add whitespace here for style reasons.

Reply:
@cygnus-x-1 fixed. I installed NetBeans and forgot to update my settings.